Palo Alto Networks/paloalto-cortex/_meta/manifest.yml

```
@@ -0,0 +1,8 @@
+uuid: 00000000-0000-0000-0000-000000000000
```

You can provide any non-existing (in this repository) uuid
```suggestion
uuid: 2652d6b3-242b-4561-9b34-4896f75d3770
```